scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
+            /* hotkey[0] => Category, hotkey[1] => Description, hotkey[2] => Shortcut, hotkey[3] => Options */
+
+            this.bind(hotkey[2], function (e) {
```

Comment:
Maybe I can add check if these shortcut is already defined, and if it is, just ignore function. Is this case have a problem with leaking memory?